bazel_codegen/lib/src/assets/file_system.dart

```diff
@@ -32,6 +33,17 @@ class BazelFileSystem {
   String readAsStringSync(String path, {Encoding encoding: UTF8}) =>
       _fileForPath(path).readAsStringSync(encoding: encoding ?? UTF8);
 
+  Iterable<String> findAssets(String packagePath, Glob glob) sync* {
```

Comment:
Lets add a test for this as well, should be able to just create a real tmp dir or something with some files and provide that as a search path.